lib/Cavil/PostProcess.pm

```diff
@@ -0,0 +1,137 @@
+# Copyright (C) 2019 SUSE Linux GmbH
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package Cavil::PostProcess;
+use Mojo::Base -base;
+
+has 'hash';
+has max_line_length => 125;
+
+sub _split_find_a_good_spot {
+  my ($self, $line) = @_;
+
+  my $index  = $self->max_line_length;
+  my $length = length($line);
+  return $length if ($index > $length);
+  my %splits = (' ' => 1, ';' => 1, '{' => 1, '}' => 1, '"' => 0);
+  while ($index > $self->max_line_length * 0.7) {
```

Comment:
The repeated calls to `$self->max_line_length`  in this method are quite costly. Better assign the value to a variable in the beginning.